setup.cfg

```diff
@@ -1,5 +1,56 @@
-[bdist_wheel]
-universal = 1
-
 [metadata]
+name = plumbum
+author = Tomer Filiba
+author_email = tomerfiliba@gmail.com
+license = MIT
+description = Plumbum: shell combinators library
+keywords =
+    path,
+    local,
+    remote,
+    ssh,
+    shell,
+    pipe,
+    popen,
+    process,
+    execution,
+    color,
+    cli
+url = https://plumbum.readthedocs.io
+long_description = file: README.rst
+long_description_content_type = text/x-rst
 license_file = LICENSE
+classifiers =
+    Development Status :: 5 - Production/Stable
+    License :: OSI Approved :: MIT License
+    Operating System :: Microsoft :: Windows
+    Operating System :: POSIX
+    Programming Language :: Python :: 2.6
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.4
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Topic :: Software Development :: Build Tools
+    Topic :: System :: Systems Administration
+platforms = POSIX, Windows
+provides = plumbum
+
+[options]
+packages =
```

Comment:
Is there a reason not to use automatic package discovery here?

Reply:
I can make it automatic. Should I?

Reply:
Sure. As a general rule it’s safer.